agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java

```diff
@@ -368,7 +364,7 @@ public void uninstallTsa(InstanceId instanceId, Topology topology, TerracottaSer
     if (terracottaInstall != null) {
       int installationsCount = terracottaInstall.removeServer(terracottaServer);
       if (installationsCount == 0) {
-        File installLocation = terracottaInstall.getRootInstallLocation();
+        File installLocation = Agent.WORK_DIR.resolve(instanceId.toString()).toFile();
```

Comment:
this does not feel right: this line should be inside the `if (kitInstallationPath == null) {` I think... It is only the remote use case

Reply:
I want the working directory of individual servers independent from the kit installation.  Each server has it's own home defined in the platform changes.

Reply:
i think i mis-understood the original point.  let me examine.

Reply:
fixed